recipes/dpu-utils/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "dpu-utils" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dpu_utils-{{ version }}.tar.gz
+  sha256: d385083cb6dde43d9f469dc8627dfd13f25d95789890b51b55f90ebeefa15311
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  skip: True  # [py>37]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - azure-identity
```

Comment:
Why is this list much longer than the one at https://github.com/microsoft/dpu-utils/blob/1ded77b6cfbe21ccd9e4544ed94297b1c05bad3a/python/setup.py#L22 ?

Reply:
Because there were some underlying libraries that needed to be installed, and they weren't getting picked up by conda correctly when `noarch` was disabled.

Reply:
This sounds weird. Can you retry with `noarch` and post the error here so it doesn't get lost? This should work as `noarch`, happy to help you get there. `noarch` packages are a lot easier to maintain.

Reply:
Alright, I am adding this in my next commit. Resolving this conversation, for now.